bpf/lib/conntrack.h

```diff
@@ -394,6 +396,9 @@ static inline int __inline__ ct_create6(void *map, struct ipv6_ct_tuple *tuple,
 		entry.tx_bytes = skb->len;
 	}
 
+	if (skb->tc_index)
+		entry.nat46 = 1;
```

Comment:
Can we use a flag bit in skb->cb instead? If we can, we should preserve tc_index as the skb passes through us. I realize this is not new code but we are converting this from prototype to real now and should clean this up.